torchopt/_src/utils.py

```diff
@@ -15,7 +15,7 @@
 
 from typing import Dict, List, NamedTuple, Union
 
-import jax
+import jax.tree_util as pytree
```

Comment:
TODO: change this to `import torch.utils.pytree as pytree` when `torch.utils.pytree` becomes GA.